src/main/kotlin/src/racingcar/domain/Winner.kt

```diff
@@ -0,0 +1,9 @@
+package src.racingcar.domain
+
+class Winner(
+    private val racingCars: List<Car>
+) {
+    fun findWinners(): List<Car> {
+        return racingCars.filter { car -> car.getMoveCount() >= racingCars.maxOf { it.getMoveCount() } }
```

Comment:
```suggestion
        val max = racingCars.maxOf { it.getMoveCount() }
        return racingCars.filter { car -> car.getMoveCount() >= max }
```
maxOf 사용 좋습니다 👍
변경 전과 변경 후의 코드의 차이는 무엇일까요?